code-reviews/README.md

```diff
@@ -157,11 +160,11 @@ For more information, see links under [resources](#resources).
 
 ## Resources
 
+* [Tools](tools.md)
```

Comment:
I'd prefer the full title in the link: Code review tools

Reply:
+1

Reply:
Changed. Thanks.